content/blog/posts/o3de-motion-matching-2022-04.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Motion Matching in O3DE, a Data-Driven Animation Technique"
+date: 2022-05-12
+slug: blog-motionmatching
+author: Benjamin Jillich
+blog_img: "/images/blog/motion-matching/blog_img.png"
+full_img: "/images/blog/motion-matching/full_image.png"
+---
+
+Motion matching is a data-driven animation technique that is gaining popularity since games like 'The Last Of Us Part II', 'FIFA', and 'Half-Life: Alyx' have adotped it. The complexity of animation graphs has increased along with higher expectations, a greater variety of interactions between characters and their environments, and the need for more realistic animations. Motion matching is a way to animate a character in a controlled way by jumping and blending between pre-recorded animation sequences several times a second, synthesizing the animation for the character. It can be used for realistic locomotion without the need to manually create carefully timed transition points and synchronization between animation clips.
+
+We brought motion matching to O3DE's animation system EMotion FX as a blend tree node so that it can still be used and mixed with everything else available in the animation graph.
+
+{{< video src="/images/blog/motion-matching/controlled-character.mp4" autoplay="true" loop="true" width="100%" info="Gamepad controlled character using motion matching." >}}
+<br>
+<br>
+
+## Motion Database
+
+To animate a character using motion matching, we need a large library of character animations such as walking slowly, running fast, narrow and wide turns, accelerating and decelerating, and many more. A large variety of movements are required so that there is a piece of animation for every situation. Kristjan Zadziuk and his team ([GDC 2016 Talk, Animation Bootcamp: Motion Matching](https://www.youtube.com/watch?v=_Bd2T7uP9VA)) gave this some structure and worked out a set of *dance cards*. These dance cards describe paths actors move along in a motion capture studio that capture the full range of motions for best visual results. They also minimize variations and redundant data that decrease the algorithm's performance.
```

Comment:
Nit: Maybe set some expectations about if we are providing a sample Motion Database ie "You can see a sample database of moves in.."? Or you will need to generate such a database, see our recommendations ... etc.

Not sure if this covers how users should store motions or set things up or where to look next.

Reply:
Added